engine/src/main/java/org/hibernate/search/util/impl/ReflectionHelper.java

```diff
@@ -115,13 +115,34 @@ public static Object getMemberValue(Object bean, XMember getter) {
 	}
 
 	/**
-	 * Checks whether the specified class contains any Search specific annotations.
+	 * Checks whether the specified class contains any Search-specific annotations, i.e. if
+	 * the class itself or one of its superclasses carries such an annotation.
```

Comment:
"Contains" seemed especially fuzzy to me. It's okay for a method name, but here in the javadoc I wouldn't want any ambiguity. I'll try something else...